introduction.md

```diff
@@ -1,2 +1,6 @@
 This is my introduction to dojo_rules.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

Reply:
I am glad you are writting